hypothesis-python/src/hypothesis/extra/django/_impl.py

```diff
@@ -81,10 +85,25 @@ def from_model(
     :obj:`~hypothesis.infer` as a keyword argument to infer a strategy for
     a field which has a default value instead of using the default.
     """
-    if not issubclass(model, dm.Model):
+    if len(model) == 1:
+        m_type = model[0]
+    elif len(model) > 1:
+        raise TypeError("Too many positional arguments")
+    else:
+        try:
+            m_type = field_strategies.pop("model")  # type: ignore
+        except KeyError:
+            raise TypeError("Must pass `model` as a positional argument") from None
```

Comment:
This error message is kinda confusing. How about "Missing required argument `model`"?